cardano/src/block/verify.rs

```diff
@@ -323,18 +323,18 @@ impl Verify for VssCertificates {
     fn verify(&self, protocol_magic: ProtocolMagic) -> Result<(), Error> {
         // check that there are no duplicate VSS keys
         let mut vss_keys = BTreeSet::new();
-        if !self.iter().all(|x| vss_keys.insert(x.vss_key.clone())) {
+        if !self.0.iter().all(|x| vss_keys.insert(x.vss_key.clone())) {
```

Comment:
I don't like all these `.0` accessors. If a wrapper type has a purpose (i.e. it's not just a meaningless newtype over something that everybody would actually want to use anyway), it should hide its contents and provide a named accessor method or methods to the inner value, `as`-style if accessing by reference, `into`-style if consumed to get the inner object.